visualizer/index.html

```diff
@@ -165,6 +166,50 @@
       color: white;
     }
 
+    #actions {
+      display: flex;
+      flex: 1;
+      position: fixed;
+      right: 20px;
+      margin-top: 40px;
+    }
+    #actions .actionColumn {
+      margin-left: 15px;
+    }
+    #actions .actionEntries {
+      flex-direction: column;
+      align-self: flex-start;
+      background-color: white;
+      color: black;
+      font-family: Menlo, Monaco, sans-serif;
+      font-style: normal;
+      border-radius: 3px;
+      padding: 2px 0px 1px 0px;
+    }
+
+    textarea.action {
+      height: 12px;
+      resize: none;
+      outline: none;
+      padding-top: 1px;
+      padding-bottom: 1px;
+      margin-bottom: 4px;
+      height: 16px;
+      border: 1px solid #555;
+      border-radius: 10px;
+      display:block;
+      clear:both;
```

Comment:
I don't think you need this. `clear` is usually only used with floated elements.